sechub-pds-solutions/findsecuritybugs/env-initial

```diff
@@ -7,4 +7,5 @@ JAVA_ENABLE_DEBUG=false
 KEEP_CONTAINER_ALIVE_AFTER_PDS_CRASHED=false
```

Comment:
This file should be removed. For more information have a look at: https://github.com/mercedes-benz/sechub/pull/1862#issuecomment-1382224019.